src/Components/user.tsx

```diff
@@ -1,16 +1,21 @@
 import React, {useState} from 'react';
 import {View, Text, StyleSheet, TouchableOpacity, Image} from 'react-native';
 import Colors from '../Contants/Colors';
-//@ts-ignore
-import UserAvatar from 'react-native-user-avatar';
 import {mvs} from '../Config/metrices';
+import defaultProfilePicture from '../Contants/defaultPicture';
 
 const User = ({item, category, onPress, ...props}: any) => {
   return (
     <TouchableOpacity onPress={onPress} style={styles.container}>
       <View>
-        {/* <Image source={{uri: item?.displayPictureUrl}} style={styles.image} /> */}
-        <UserAvatar size={50} name={item.fullName} />
+        <Image
+          source={{
+            uri: item?.profilePicture
```

Comment:
use this:
uri:  item?.profilePicture | | defaultProfilePicture
instead of this:
uri: item?.profilePicture ?  item?.profilePicture : defaultProfilePicture,